test/parse/test.sh

```
@@ -0,0 +1,22 @@
+    ["rt_file/error_A_duplicate.rt"]=1
+    ["rt_file/error_A_char.rt"]=1
+    ["rt_file/success.rt"]=0
+)
+
+for file in "${!test_cases[@]}"; do
+    expected_status="${test_cases[$file]}"
+    ./parse "$file"
+    if [ "$?" -eq "$expected_status" ]; then
+        echo -e "\e[32mOK\e[0m"
+    else
+        echo -e "\e[31mNG (got $status)\e[0m"
```

Shellcheck (0.10.0)

[warning] status is referenced but not assigned.

(SC2154)